src/main/java/app/coronawarn/verification/portal/controller/VerificationPortalController.java

```diff
@@ -90,8 +90,11 @@
   /**
    * The REST client interface for getting the TeleTAN from verificationserver.
    */
-  @Autowired
-  private TeleTanService teleTanService;
```

Comment:
Thx for ur constribution. I agree about distaste for field injection, but finally I would recommend the usage of @RequiredArgsConstructor annotation - will cause a constructor being added during the compilation process taking all final fields as parameters and the additional @NonNull annotation will cause the parameter be checked for null as well.